third_party/move/move-compiler/src/expansion/translate.rs

```diff
@@ -384,6 +394,32 @@ fn set_sender_address(
     })
 }
 
+// This is a hack to recognize APTOS StdLib to avoid warnings on some old errors.
+// This will be removed after library attributes are cleaned up.
+// (See https://github.com/aptos-labs/aptos-core/issues/9410)
+fn module_is_in_aptos_stdlib(module_address: Option<Spanned<Address>>) -> bool {
```

Comment:
maybe we should just relabel this to "stdlib" but I guess this just exists temporarily anyway...

Reply:
Thx.